Automattic/jetpack
# Likes: move editor plugin to the plugins' directory.

> **Warning**
> Let's put this on hold to avoid rebasing fun; we can resume once #33163 will be fixed.

## Proposed changes:

Just like in #33238 and #33240, let's move this editor plugin to its right location in the codebase.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Existing functionality should not change.

* Go to Jetpack > Settings 
* Search and enable Likes
* Go to Posts > Add New
* Open the Jetpack plugin sidebar
* You should find the Likes panel, and it should continue to work.
